plutus-core/plutus-core/stdlib/PlutusCore/StdLib/Data/MatchOption.hs

```diff
@@ -2,9 +2,8 @@ module PlutusCore.StdLib.Data.MatchOption
     ( MatchOption (..)
     ) where
 
--- | Allows one to choose which way of doing pattern matching on built-in types to use: either via
--- 'ChooseList'-like builtins or via 'CaseList'-like ones.
+-- | Allows one to choose which way of doing pattern matching on built-in types to use: currently
+-- only 'ChooseList'-like builtins are supported.
 data MatchOption
```

Comment:
Do you plan to add something else here soon?  (Just wondering why you're leaving it in if there's only one possibility.)